### PR TITLE
Update readme 

### PR DESCRIPTION
**Pull request by kirtangajjar**
Signed-off-by: Kirtan Gajjar <kirtangajjar95@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
-# cron-command
+# EasyEngine/cron-command
+
+Manages cron jobs in EasyEngine
+
+`cron` command contains following subcommand
+ * [add](#add)
```
Comment (mbtamuli) [inline README.md]: Mention the full command like https://github.com/EasyEngine/site-command/blob/develop/README.md to maintain similarity

File: README.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+ * [list](#list)
+ * [delete](#delete)
+ * [run-now](#run-now)
+ 
+ ## add
+ 
+ Adds a cron job to run a command at specific interval etc.
+
+ ```
+ # Adds a cron job on example.com every 10 minutes
```
Comment (mbtamuli) [inline README.md]: We should give one more example explaining traditional cron schedule is also supported. For example, for every one minute `* * * * *`

File: README.md [hunk 3]
```diff
@@ -1 +1,97 @@
...
+ # Updates site to run cron on
+ $ ee cron update 1 --site='example1.com'
+ 
+ # Updates command of cron
+ $ ee cron update 1 --command='wp cron event run --due-now'
+ 
+ # Updates schedule of cron
+ $ ee cron update 1 --schedule='@every 1m'
+ ```
+  git Also, refer to [possible schedule values](#possible-schedule-values) to know more about it.
```
Comment (mbtamuli) [inline README.md]: Please correct the typo at this line.

File: README.md (hunk 1 above)
Comment (kirtangajjar) [reply README.md]: @mbtamuli It was getting redundant. So I thought it's best to keep it short like this and update it like this in site-command too

File: README.md (hunk 2 above)
Comment (kirtangajjar) [reply README.md]: Good idea. Done

File: README.md (hunk 1 above)
Comment (mbtamuli) [reply README.md]: > update it like this in site-command too
Okay

**Pull request merged**